go.mod

```diff
@@ -3,14 +3,37 @@ module github.com/chef/go-chef
 go 1.12
 
 require (
+	github.com/aws/aws-sdk-go v1.30.7 // indirect
```

Comment:
Most of these dependencies are pulled in for the integration tests. I would love to have those tests pull their own dependencies in with their own `go.mod` but I think we fix that in https://github.com/chef/go-chef/issues/15

Reply:
Sounds good. Thanks @tyler-ball